src/background/Wallet/Wallet.ts

```diff
@@ -92,7 +101,38 @@ import {
   ReadonlyAccountContainer,
 } from './model/AccountContainer';
 
-async function prepareNonce<T extends { nonce?: number; from?: string }>(
+if (FEATURE_PAYMASTER_ENABLED) {
+  Object.assign(globalThis, { EIP712Signer, zkSyncUtils });
+}
+
+function createTypedData({
```

Comment:
May be this can be moved to another file, just to avoid expanding Wallet.ts :)

Reply:
Sure! Once we have a better feel for the abstraction